docker-compose.yaml

```diff
@@ -33,7 +33,19 @@ services:
       - postgres
     ports:
       - 5433:5433
-    command: ["--connection", "postgres://postgres:password@postgres:5432/postgres?sslmode=disable", "--port", "5433", "--schema", "mergestat,public", "--enhance-graphiql", "--allow-explain", "--append-plugins", "@graphile-contrib/pg-simplify-inflector", "--no-ignore-rbac"]
+    command:
+      [
+        "--connection",
+        "postgres://postgres:password@postgres:5432/postgres?sslmode=disable",
+        "--port",
+        "5433",
+        "--schema",
+        "mergestat,public",
+        "--enhance-graphiql",
+        "--allow-explain",
+        "--append-plugins",
+        "--no-ignore-rbac",
+      ]
```

Comment:
note that this fails to start as the `--append-plugins` flag expects a list of plugins right after it. If we are not going to use the `pg-simplify-inflector`, we should remove this flag completely. Otherwise it should be `--append-plugins @graphile-contrib/pg-simplify-inflector`

Reply:
I left  `@graphile-contrib/pg-simplify-inflector` plugin, yes I am agree, it produces more readable queries. Done here [f7e31e3](https://github.com/mergestat/fuse/pull/74/commits/f7e31e34096ffb956c46476efc0b479e3c4be4c1)